### ISSUE TITLE
Document IPFS storage location

### ISSUE THREAD

**Issue by abitrolly** [elopio/ipfs-snap#14]
There is no `Datastore.Path` in `ipfs config show` as described in the last step of this tutorial:

https://flyingzumwalt.gitbooks.io/decentralized-web-primer/install-ipfs/lessons/initialize-repository.html

I'd like to see all files that are served by my node.

**Comment by abitrolly** [comment]
Found some files in:
```
ls -la ~/snap/ipfs/current/.ipfs
```
Maybe gitbooks should be updated?

**Comment by come-maiz** [comment]
@abitrolly that document doesn't install the IPFS snap.

**Comment by abitrolly** [comment]
@elopio yes, it doesn't. Bit shouldn't `snap` behaviour be the same for user regardless of installation method? At least when it is about user data.

**Comment by mkg20001** [comment]
@abitrolly Problem is that snaps can't access dotfiles outside their own data directory, so even with the `home` permission the snap couldn't access `$HOME/.ipfs`

**Comment by come-maiz** [comment]
@abitrolly I would say no. snap is a new technology that's trying to implement safer ways to share files between apps. It's impossible to implement that change while preserving the past behavior. But this should be discussed https://forum.snapcraft.io/

**Comment by abitrolly** [comment]
Is it possible to patch `ipfs` to give correct info on `Datastore.Path` in `ipfs config show`?

Interesting if it is possible to extract `Datastore.Path` with a single command. Filled https://github.com/ipfs/go-ipfs/issues/5603 for that.

**Comment by jcalfee** [comment]
Where is `ipfs add <path>` looking?
